mindsdb_native/libs/backends/lightwood.py

```diff
@@ -318,6 +320,14 @@ def train(self):
                 mixer_classes = [use_mixers]
         else:
             mixer_classes = lightwood.mixers.BaseMixer.__subclasses__()
+            if self.transaction.lmd['stop_training_in_x_seconds'] is not None:
+                if self.transaction.lmd['stop_training_in_x_seconds'] > reasonable_training_time:
+                    mixer_classes = lightwood.mixers.BaseMixer.__subclasses__()
+                elif reasonable_training_time / 10 < self.transaction.lmd['stop_training_in_x_seconds'] < reasonable_training_time:
+                    mixer_classes = [lightwood.mixers.NnMixer]
+                elif reasonable_training_time / 10 > self.transaction.lmd['stop_training_in_x_seconds']:
```

Comment:
Your debug flag comes in handy here, only do this in "debug" mode, aka when testing, but not when we release.

Reply:
Actually, probably better to just comment this out from now, and let's get back to it after the Precona release.